pkg/executor/insert_test.go

```diff
@@ -592,3 +593,30 @@ func TestInsertLockUnchangedKeys(t *testing.T) {
 		}
 	}
 }
+
+func TestInsertNullInNonStrictMode(t *testing.T) {
```

Comment:
Could you please move(rewrite) this case to ['integration tests'](https://github.com/pingcap/tidb/tree/master/tests/integrationtest)? Thanks.